### ISSUE TITLE
Undismissable palletes in Dollar street

### ISSUE THREAD

**Issue by VishnuVardhanBR** [llaske/sugarizer#1605]
Palettes are undismissable when opened and then a item is opened.

![dollarstreet](https://github.com/llaske/sugarizer/assets/64434937/de401f0f-4b8b-4170-b2d7-7090ffcd535b)

**Comment by llaske** [comment]
You're right. It could be logical to close the palette automatically since the detail is a new screen.

**Comment by llaske** [comment]
Fixed in https://github.com/llaske/sugarizer/pull/1610
